projects/plugins/backup/src/js/components/Backups.js

```diff
@@ -0,0 +1,351 @@
+								width="248"
+								height="200"
+								filterUnits="userSpaceOnUse"
+								color-interpolation-filters="sRGB"
+							>
+								<feFlood flood-opacity="0" result="BackgroundImageFix"></feFlood>
+								<feColorMatrix
+									in="SourceAlpha"
+									values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
+								></feColorMatrix>
+								<feOffset></feOffset>
+								<feGaussianBlur stdDeviation="20"></feGaussianBlur>
+								<feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.08 0"></feColorMatrix>
+								<feBlend in2="BackgroundImageFix" result="effect2_dropShadow"></feBlend>
+								<feBlend in="SourceGraphic" in2="effect2_dropShadow" result="shape"></feBlend>
+							</filter>
+						</defs>
+					</svg>
+
+					<svg
+						class="backup__animation-el-3"
+						width="536"
+						height="196"
+						fill="none"
+						xmlns="http://www.w3.org/2000/svg"
+					>
+						<g filter="url(#filter3_d)">
+							<rect x="40" y="40" width="456" height="116" rx="8" fill="#fff"></rect>
+						</g>
+						<path
+							d="M475.35 62.04A7.49 7.49 0 00468 56c-2.89 0-5.4 1.64-6.65 4.04A5.994 5.994 0 00456 66c0 3.31 2.69 6 6 6h13c2.76 0 5-2.24 5-5 0-2.64-2.05-4.78-4.65-4.96z"
+							fill="#E9EFF5"
+						></path>
+						<circle cx="100" cy="98" r="36" fill="#F7A8C3"></circle>
+						<path
+							d="M160 84a6 6 0 016-6h174a6 6 0 110 12H166a6 6 0 01-6-6zM160 112a6 6 0 016-6h276a6 6 0 110 12H166a6 6 0 01-6-6z"
+							fill="#E9EFF5"
+						></path>
+						<defs>
+							<filter
+								id="filter3_d"
+								x="0"
+								y="0"
+								width="536"
+								height="196"
+								filterUnits="userSpaceOnUse"
+								color-interpolation-filters="sRGB"
+							>
+								<feFlood flood-opacity="0" result="BackgroundImageFix"></feFlood>
+								<feColorMatrix
+									in="SourceAlpha"
+									values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
+								></feColorMatrix>
+								<feOffset></feOffset>
+								<feGaussianBlur stdDeviation="20"></feGaussianBlur>
+								<feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.08 0"></feColorMatrix>
+								<feBlend in2="BackgroundImageFix" result="effect3_dropShadow"></feBlend>
+								<feBlend in="SourceGraphic" in2="effect3_dropShadow" result="shape"></feBlend>
+							</filter>
+						</defs>
+					</svg>
+				</div>
+			</div>
+		);
+	};
+
+	const formatDateString = dateString => {
+		const todayString = __( 'Today', 'jetpack-backup' );
+		const todayDate = getDate();
+		let backupDate = todayString;
+		if ( dateI18n( 'zY', todayDate ) !== dateI18n( 'zY', dateString ) ) {
+			backupDate = dateI18n( 'M j', dateString );
+		}
+		const backupTime = dateI18n( 'g:i A', dateString );
+
+		return backupDate + ', ' + backupTime;
+	};
+
+	const renderCompleteBackup = () => {
+		return (
+			<div className="jp-row">
+				<div className="lg-col-span-3 md-col-span-4 sm-col-span-4">
+					<div className="backup__latest">
+						<img src={ CloudIcon } alt="Jetpack Logo" />
```

Comment:
You'll probably want to translate this alt text as well.

Reply:
Wrong alt text there anyway.  I'll probably make it an empty string since it's a simple decorative image.